browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -411,6 +418,15 @@ void BraveBrowserView::AddedToWidget() {
         VerticalTabStripWidgetDelegateView::Create(
             this, vertical_tab_strip_host_view_);
 
+    // By setting this property to the widget for vertical tabs,
```

Comment:
This does not sound like a good idea to me. It seems like there is a problem with how we've implemented the tab strip if this is even necessary. Where exactly is this a problem?